registry/glossary.md

```diff
@@ -1,68 +1,4 @@
 ---
-published: false
+redirect_to: https://github.com/distribution/distribution/tree/main/docs/glossary.md
```

Comment:
Looks like we didn't publish this page (already a 404), so we can remove this file altogether.